spec/controllers/search_spec.rb

```diff
@@ -5,7 +5,8 @@
   describe "POST #slack", vcr: true do
     
     let(:team) { FactoryGirl.create(:team) }
-    let!(:post_params) { post_params = {text: "dude", channel_name: "testing_reactif", user_name: Faker::Internet.user_name, team_domain: team.domain}}
+    let(:channel_name) { "testing_reactif" }
+    let!(:post_params) { post_params = {text: "dude", channel_name: channel_name, user_name: Faker::Internet.user_name, team_domain: team.domain}}
```

Comment:
Useless assignment to variable - `post_params`.<br>Line is too long. [146/80]<br>Space missing inside }.<br>Space inside { missing.<br>Space inside } missing.